editor/js/Viewport.Info.js

```diff
@@ -98,6 +105,30 @@ function ViewportInfo( editor ) {
 
 	}
 
+	//
+
+	editor.signals.pathTracerUpdated.add( function ( samples ) {
+
+		samples = Math.floor( samples );
+
+		samplesText.setValue( samples );
+
+		const samplesStringKey = ( pluralRules.select( samples ) === 'one' ) ? 'viewport/info/oneSample' : 'viewport/info/samples';
```

Comment:
It seems the plural logic is still in place though....

Reply:
> Do you mind leaving the plural refactoring for another PR?

^^^^^^^

:D

Reply:
I think the current plural code is fine (plus the clean up).

The refactoring was over engineering.